doc/changelog/04-tactics/13237-master+fix13235-no-degenerate-in-hyps-clause.rst

```diff
@@ -0,0 +1,5 @@
+- **Changed:**
+  Prevent the :n:`in @goal_occurrences` clause of tactics to silently be empty
```

Comment:
```suggestion
  Require at least one occurrence in TBD
```

Better to state this positively.  Can't use `goal_occurrences`, there's no such nonterminal.

Reply:
FYI tactics.rst hasn't been revised.  That definition isn't come from doc_grammar.  FWIW #13343 will update the occurrences grammar.